esp32/sdkconfig.h

```diff
@@ -36,7 +44,6 @@
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK 0
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK_CPU1 0
 
-#define CONFIG_FREERTOS_UNICORE 1
```

Comment:
> Try adding -Wno-error=array-bounds to CFLAGS for the dport_access.c file.

I'll try it, but that function is unconditionally a part of the IDF, however, I don't know if anything links to it.

The example I used was the `gatt_server_service_table`, though all I did was let it write the defaults for `sdkconfig`.